libdevcore/CommonData.h

```diff
@@ -172,9 +179,9 @@ inline std::string toHex(u256 val, HexPrefix prefix = HexPrefix::DontAdd)
 	return (prefix == HexPrefix::Add) ? "0x" + str : str;
 }
 
-inline std::string toCompactHex(u256 val, HexPrefix prefix = HexPrefix::DontAdd, unsigned _min = 0)
+inline std::string toCompactHex(u256 val, HexPrefix prefix = HexPrefix::DontAdd, PrintZero _min = PrintZero::DontPrint)
```

Comment:
So many places where we pass `PrintZero::Print` here, is it ever used really with `PrintZero::DontPrint` ?